thousandeyes/util_test.go

```diff
@@ -317,11 +320,19 @@ func TestFixReadValues(t *testing.T) {
 			"test_id":   "2",
 		},
 	}
+	testsTarget := []interface{}{
+		map[string]interface{}{
+			"test_id": "1",
+		},
+		map[string]interface{}{
+			"test_id": "2",
+		},
+	}
 	output, err = FixReadValues(testsInput, "tests")
 	if err != nil {
 		t.Errorf("tests input returned error: %s", err.Error())
 	}
-	if output != nil {
+	if reflect.DeepEqual(output, testsTarget) != true {
```

Comment:
Just fixing this, nothing changed as part of this PR. This was was breaking for a long time already. For the tests we do nothing, so the output should be the same as the input